tools/syz-declextract/run.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2024 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"os"
+	"os/exec"
+	"runtime"
+	"strings"
+
+	"github.com/google/syzkaller/pkg/tool"
+)
+
+type compileCommand struct {
+	Arguments []string
+	Directory string
+	File      string
+	Output    string
+}
+
+type output struct {
+	stdout []byte
+	stderr []byte
+}
+
+func main() {
+	compilationDatabase := flag.String("p", "compile_commands.json", "Path to compilation database")
+	binary := flag.String("b", "syz-declextract", "Path to binary")
+	flag.Parse()
+
+	fileData, err := os.ReadFile(*compilationDatabase)
+
```

Comment:
remove the empty line

generally, statement that creates `err` and checks err are not separated.